Bottom: c26d863c5dc11498a04a7eec07edaff64c2bbe82
Top:    33678e58443e541c9a8cf3b20b547ba134bdb405
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-05 11:19:43 +0800

Refresh of add-a-vq-meta-data-iotlb

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 778af6f..9ac1c1c 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -864,7 +864,7 @@ static void __user *__vhost_get_user(struct vhost_virtqueue *vq,
 	} else { \
 		__typeof__(ptr) to = \
 			(__typeof__(ptr)) __vhost_get_user(vq, ptr,	\
-							sizeof(*ptr), type); \
+							   sizeof(*ptr), type); \
 		if (to != NULL) \
 			ret = __put_user(x, to); \
 		else \
@@ -880,7 +880,8 @@ static void __user *__vhost_get_user(struct vhost_virtqueue *vq,
 		ret = __get_user(x, ptr); \
 	} else { \
 		__typeof__(ptr) from = \
-			(__typeof__(ptr)) __vhost_get_user(vq, ptr, sizeof(*ptr),type); \
+			(__typeof__(ptr)) __vhost_get_user(vq, ptr,
+							   sizeof(*ptr),type); \
 		if (from != NULL) \
 			ret = __get_user(x, from); \
 		else \
